src/Toml/Edsl.hs

```diff
@@ -9,20 +9,28 @@ exampleToml = mkToml $ do
     \"key2\" =: Bool True
     table \"tableName\" $
         \"tableKey\" =: Array [\"Oh\", \"Hi\", \"Mark\"]
+    array \"arrayName\" $
```

Comment:
I guess, it's also should be `tableArray` now?

Reply:
Of course, thank you!